docs/guides/trace/sni.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Using trace sni'
+weight: 10
+---
+
+The `trace sni` gadget is used to trace the [Server Name Indication (SNI)](https://en.wikipedia.org/wiki/Server_Name_Indication) requests sent as part of TLS handshakes.
+
+## How to use it?
+
+The SNI tracer will show which pods are making which SNI requests. To start it,
+we can run:
+
+```bash
+$ kubectl gadget trace sni
+POD                            NAME
+```
+
+To generate some output for this example, let's create a demo pod in *another terminal*:
+
+```bash
+$ kubectl run -it ubuntu --image ubuntu:latest -- /bin/bash
+root@ubuntu:/# apt update && apt install -y wget && wget wikimedia.org
+(...)
+HTTP request sent, awaiting response... 301 Moved Permanently
+Location: https://www.wikimedia.org/ [following]
+(...)
+root@ubuntu:/# wget www.github.io
```

Comment:
nit: is it really `www.github.io` or should it be `www.github.com`?

When I run `wget www.github.io` I get a SNI request for `pages.github.com`  instead of `github.com`.